deployment/src/main/resources/templates/api.qute

```diff
@@ -53,6 +53,11 @@ public interface {classname} {
     {#if op.hasProduces}
     @Produces(\{{#for produce in op.produces}"{produce.mediaType}"{#if produce_hasNext}, {/if}{/for}\})
     {/if}
+    {#for cbClassConfig in circuit-breaker.classConfigurations.orEmpty}{#if cbClassConfig.className == package + classname}
```

Comment:
If using a map, I think this will still work, you just need to change
cbClassConfig.className by cbClassConfig.key
and
cbClassConfig.methods by cbClassConfig.value